source/_components/plugwise.markdown

```diff
@@ -0,0 +1,156 @@
+---
+title: "Plugwise"
+description: "Plugwise Climate integration."
+logo: plugwise.png
+ha_category: Climate
+ha_iot_class: Local Polling
+ha_release: 0.97
+---
+
+This enables [Plugwise](https://plugwise.com) [Anna](https://www.plugwise.com/en_US/products/anna) thermostats to be integrated. This integration talks locally to your **Smile** interface, you will need it's secret key and IP address.
+
+To find your IP address using the Plugwise App. When connected choose the 'Settings'-icon and go to 'HTML-interface'. Go to the (now little lower) 'Settings'-icon and choose 'Preferences'. Under 'System' then 'Networking' you will find your IP address.
+
+## Configuration
+
+You have to add the following to your configuration.yaml file.
+
+```yaml
+# Minimal configuration.yaml entry
+climate:
+  - platform: plugwise
+    password: your_short_id # required, the ID on the smile (some string of 6 characters)
+    host: local_ip_address  # required, the IP-address of your smile
+```
+
+{% configuration %}
+password:
+  description: Your Smile ID (located on the bottom of the Smile, not the Anna)
+  required: true
+  type: string
+host:
+  description: The IP address of your Smile. 
+  required: true
+  type: string
+name:
+  description: The name of your thermostat, i.e. "Anna"
+  required: false
+  type: string
+username:
+  description: Should you ever need to change this, you can
+  required: false
```

Comment:
Add the default value using the `default` key.

Reply:
Added all the defaults accordingly